Tribler/Core/Modules/restapi/downloads_endpoint.py

```diff
@@ -94,11 +96,12 @@ def render_GET(self, request):
                              "speed_down": download.get_current_speed(DOWNLOAD),
                              "speed_up": download.get_current_speed(UPLOAD),
                              "status": dlstatus_strings[download.get_status()],
-                             "size": download.get_length(), "eta": download.network_calc_eta(),
+                             "size": download.get_def().get_length(), "eta": download.network_calc_eta(),
```

Comment:
I agree, it's a bit more readable with a newline here indeed, will add it when I process @whirm 's feedback